examples/fingerprint_picturetest.py

```diff
@@ -0,0 +1,254 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
+# SPDX-License-Identifier: MIT
+
+# Added 'View Print' and 'Preview and Find Print' functions to
+# example code fingerprint_simpletest.py
+
+import time
+
+# import board
+# import busio
+# from digitalio import DigitalInOut, Direction
+import numpy as np
+from matplotlib import pyplot as plt
+import serial
```

Comment:
Since these are used in examples, we need to add these as dependencies to `optional_requirements.txt`!